mmcv/data_pipeline/data_pipeline.py

```diff
@@ -0,0 +1,69 @@
+from collections.abc import Sequence
+
+from mmcv.utils import build_from_cfg
+from .base_transform import TRANSFORMS, BaseTransform
+
+
+class DataPipeline:
+
+    def __init__(self, transforms):
+        if not isinstance(transforms, Sequence):
+            raise TypeError('transforms must be a sequence')
+        self.transforms = []
+        for transform in transforms:
+            if isinstance(transform, dict):
+                transform = build_from_cfg(transform, TRANSFORMS)
```

Comment:
Maybe we could still create a method `build_transform` and use that to create transforms.